src/devhostagent/devhostAgent.csproj

```diff
@@ -34,8 +34,6 @@
 
   <ItemGroup>
     <PackageReference Include="Autofac.Extensions.DependencyInjection" Version="8.0.0" />
-    <PackageReference Include="Microsoft.AspNetCore.SignalR.Protocols.MessagePack" Version="6.0.8" />

```

Comment:
Any idea why signalR packages were removed ? Was this intentional ? Please let us know.Thanks.

Reply:
Yes that was intentionally. SignalR packages was already included in the dependent common project.

I couldn't find any documentation describing the desired way of handling references to NuGet packages in project files so I went for the "first project in dependency graph that uses a package have the dependency".

I'm also fine with "any project using a dependency have the dependency" approach, just let me know how you want it and I'll fix it :)